### DISCUSSION TITLE
[Avalon] Need active menu state, even after refresh

### DISCUSSION THREAD

**Discussion by ace-featherlight** [primefaces/community#30] [PrimeVue Templates]
Can the template fix the active menu state ?
Currently when I click the menu, the active state is correct
![image](https://github.com/primefaces/community/assets/78999773/1c643612-6a34-4a3a-94a0-de85a5237927)
but when I refresh the browser on same page, the active menu just gone
![image](https://github.com/primefaces/community/assets/78999773/85c45b5d-ff1d-4cd7-895a-5a73bcc588fc)

I try another avalon template, (avalon react template)
the active menu still there, even after refresh
![image](https://github.com/primefaces/community/assets/78999773/a484fdc2-babf-4832-aa89-01de364ccc91)

**Comment by sezisfurkan** [comment]
This issue will be fixed on new version.

**Reply by ace-featherlight** [reply]
Is the themes have github repo ?
So we can know and track what the code changes if have new version
I think will be hard if I use the template, the application grow up, we don't know what must update if we want to fixed some issue from the template

Thanks

**Reply by cagataycivici** [reply]
The source repo is private so I'm afraid issue tracker is private as well, however we generate a public changelog. @sezisfurkan olsa check other templates, updates are due soon so please make sure this goes in.

**Comment by ace-featherlight** [comment]
Thanks, fixed on new version
